CovertInfiltration/Src/CovertInfiltration/Classes/X2DownloadableContentInfo_CovertInfiltration.uc

```diff
@@ -16,6 +16,9 @@ static event InstallNewCampaign(XComGameState StartState)
 {
 	class'XComGameState_CovertInfiltrationInfo'.static.CreateInfo(StartState);
 	CreateGoldenPathActions(StartState);
+
+	class'XComGameState_Objective'.static.CompleteObjectiveByName(StartState, 'XP2_M0_FirstCovertActionTutorial');
+	class'XComGameState_Objective'.static.CompleteObjectiveByName(StartState, 'XP2_M1_SecondCovertActionTutorial');
```

Comment:
* Put it in a separate function (like the golden path actions)
* Also call it in loaded save game - we want to retain basic compatibility with adding the mod mid campaign